docs/router/cosmo-streams/custom-modules.mdx

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
This file is actually located inside of the custom-modules as `Overview`. I think it makes more sense to logically place all the files in the corresponding folders. Also maybe name it overview.mdx instead of custom-modules.mdx.

```sh
mv custom-modules.mdx ./custom-modules/overview.mdx
```

Reply:
I see